pkgs/java_http/README.md

```diff
@@ -0,0 +1,3 @@
+## Status: Experimental
```

Comment:
Could you use the same formatting as https://github.com/dart-lang/http/blob/master/pkgs/cronet_http/README.md ?

That is:
- Start with a description of the package.
- Copy the experimental section from `package:cronet_http` verbatim

Otherwise looks good!

Reply:
Hey Brian,

I made the following changes:

- Copied the "Status: Experimental" section from `package:cronet_http`'s README verbatim.

- Moved the package description so that it is above the "Status: Experimental" section as it is with `package:cronet_http`'s README.

I also updated the description to include the advantages of using java_http on Android.
This was borrowed from the [Dart GSoC 2023 Project Ideas](https://github.com/dart-lang/sdk/wiki/Dart-GSoC-2023-Project-Ideas#idea-build-a-dart-http-client-using-java-apis) wiki. What are your thoughts?

Thank you!

Reply:
Sounds good to me!